index.js

```diff
@@ -12,25 +12,15 @@ var net = require("net"),
     parsers = [], commands,
     connection_id = 0,
     default_port = 6379,
-    default_host = "127.0.0.1";
-
-// can set this to true to enable for all connections
-exports.debug_mode = false;
-
-var arraySlice = Array.prototype.slice;
-function trace() {
-    if (!exports.debug_mode) return;
-    console.log.apply(null, arraySlice.call(arguments));
-}
+    default_host = "127.0.0.1",
+    debug = util.debuglog('redis');
```

Comment:
Thx, I did not pay attention to that. I added a fallback mode for 0.10 that does just the same as the debuglog.